### PR TITLE
Ingestion: paclic-22

### PR DESCRIPTION
**Pull request by anthology-assist**

### CODE REVIEW

File: data/xml/2022.paclic.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <collection id="2022.paclic">
-  <volume id="1" ingest-date="2023-04-23">
+  <volume id="1" ingest-date="2023-07-25">
```
Comment (mjpost) [inline data/xml/2022.paclic.xml:3]: This shouldn't change?

**Comment by mjpost** [comment]
Can you explain this? Are you just removing a link to a video?

**Comment by anthology-assist** [comment]
I saw the issue https://github.com/acl-org/acl-anthology/issues/2354 and I thought the ingestion material is new. I'll investigate.

**Comment by anthology-assist** [comment]
closing because it's ingested https://aclanthology.org/volumes/2022.paclic-1/

**Pull request closed without merging**